recipe/meta.yaml

```
@@ -18,42 +18,50 @@ build:
   entry_points:
     - jirashell = jira.jirashell:main
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: python -m pip install --no-deps --no-build-isolation --ignore-installed .
 
 requirements:
-  build:
-    - python >=3.6
+  host:
     - pip
-    - pbr >=3.0
-    - setuptools >=17.1
-    - pytest-runner
-    - sphinx >=2.2.0
-    - setuptools_scm >=1.15.0
-    - setuptools_scm_git_archive >=1.0
+    - python >=3.8
+    - setuptools >=60.0.0
+    - setuptools-scm >=7.0.0
+    - wheel
 
   run:
-    - python >=3.6
     - defusedxml
     - keyring
-    - requests-oauthlib >=1.1.0
+    - packaging
+    - pillow >=2.1.0
+    - python >=3.8
     - requests >=2.10.0
+    - requests-oauthlib >=1.1.0
     - requests-toolbelt
-    - setuptools >=20.10.1
-    - ipython >=4.0.0
+    - typing_extensions >=3.7.4.2
```

```suggestion
    - typing_extensions >=3.7.4.2
    - ipython >=4.0.0
```
Because we ship the CLI (https://inspector.pypi.io/project/jira/3.6.0/packages/56/93/672099bb1ee8e6018117e55d8a4307def1fffc41df079dfaa269dfad1540/jira-3.6.0-py3-none-any.whl/jira-3.6.0.dist-info/METADATA#line.48) and https://github.com/pycontribs/jira/blob/3.6.0/jira/jirashell.py#L356.